invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DeleteCommunityModal.js

```diff
@@ -25,12 +26,22 @@ export class DeleteCommunityModal extends Component {
       checkboxSlug: false,
       inputSlug: "",
       error: undefined,
+      recordsCount: false,
     };
     this.checkboxRef = React.createRef();
     this.openModalBtnRef = React.createRef();
     this.state = this.INITIAL_STATE;
   }
 
+  componentDidMount() {
+    const { community } = this.props;
+    axios.get(`/api/communities/${community.id}/records`).then((response) => {
```

Comment:
the aync fetch should be wrapped in `withCancel` function, which helps us to avoid memory leaks in case of component being unmounted before the request finishes